core/chains/evm/txmgr/evm_tx_store.go

```diff
@@ -10,6 +10,8 @@ import (
 	"strings"
 	"time"
 
+	"github.com/smartcontractkit/chainlink/v2/core/services/pg"
```

Comment:
Does the linter complain about this? Should be grouped down below with the other chainlink/v2 lines.

Reply:
Funny the linter didn't complain. Removed this dependency anyways!